archlinuxcn/devtools-archlinuxcn/lilac.yaml

```diff
@@ -3,4 +3,10 @@
 maintainers:
   - github: farseerfc
 
-managed: false
+post_build: git_pkgbuild_commit
+
+update_on:
+  - source: manual
+    manual: 20220207
```

Comment:
Forgot to update this..... I'll leave it to the next person updating this package to worry about this in a few months....